habitat_baselines/common/base_trainer.py

```diff
@@ -97,7 +113,14 @@ def eval(self) -> None:
                 len(self.config.VIDEO_DIR) > 0
             ), "Must specify a directory for storing videos on disk"
 
-        with get_writer(self.config, flush_secs=self.flush_secs) as writer:
+        if prev_ckpt_ind == -1:
+            purge_step = 0
+        else:
+            purge_step = None
+
+        with get_writer(
+            self.config, flush_secs=self.flush_secs, purge_step=purge_step
```

Comment:
The WandB or future writer might have an error if it has this new key it does not know about. Is purge_step necessary?

Reply:
Actually backed this one out. The one for training is much more important.